### PR TITLE
Add reports to scheduled table

### LINKED ISSUE #3742: Reports not visible on scheduler tab after being generated
**Describe the bug**
When opening a row of the table on the scheduled reports page, no tasks/reports are shown. Even though the report has already been generated. 

**To Reproduce**

1. Create a scheduled report
2. Go to the Scheduled Report page
3. Edit the scheduled report (open the row and press the edit button)
4. Change the cron to "* * * * *"
5. After a minute, you should see a report on the Report History page
6. Then go to the Scheduled Report page
7. Open the row of the schedule
8. See that there's no data

**Expected behavior**
The tasks/reports should be visible. See the design:
https://www.figma.com/design/ASfstypqSx22Bw7XDbVEdo/%E2%9C%85-KAT-%7C-Approved-%7C-Reports-Page?node-id=12-32789

**Additional context**
Before changing this, please have a discussion with the OpenKAT design team about how to implement this.

### PR DESCRIPTION
**Pull request by madelondohmen**
**Note:** This PR is based on the code of PR #3769 

-------------------

### Changes
This PR fixes the reports that weren't showed on the Scheduled Report page.

### Issue link
Closes #3742 

### Demo
After a scheduled report has been generated, you will now see the generated reports in the scheduled table:

afbeelding

### QA notes
- Create a scheduled report
- Change the cron to * * * * *
- Wait a minute
- Go to Scheduled Report page
- Open the schedule table row
- You should see the generated report

---

### Code Checklist

- [ ] All the commits in this PR are properly PGP-signed and verified.
- [ ] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] I have checked the documentation and made changes where necessary.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

- [ ] Tickets have been created for newly discovered issues.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Comment by stephanie0x00** [comment]
## Checklist for QA:

- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
  - [x] I have managed to pass the onboarding flow
  - [x] Objects and Findings are created properly
  - [x] Tasks are created and completed properly
- [ ] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [ ] I checked the logs for errors and/or warnings and made issues where necessary

### What works:
- Onboarding
- Scheduled reports run. 

### What doesn't work:
Opening the scheduled report page results in an error: 

image


### Bug or feature?:
n/a

**Comment by stephanie0x00** [comment]
## Checklist for QA:

- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
  - [x] I have managed to pass the onboarding flow
  - [x] Objects and Findings are created properly
  - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [x] I checked the logs for errors and/or warnings and made issues where necessary

### What works:
Looks good. All reports are shown under the Scheduled tab under the right report recipe. Works for both the normal and the aggregate reports. Reports can also be opened from the Scheduled tab. 

### What doesn't work:
n/a

### Bug or feature?:
n/a

File: rocky/reports/views/base.py [hunk 1]
```diff
@@ -537,6 +561,24 @@ def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
...
+                    "search_string": self.search_string,
+                    "order_by": self.order_by,
+                    "asc_desc": self.sorting_order,
+                }
+
+            parent_report_type = None
+            if self.report_type == AggregateOrganisationReport:
+                parent_report_type = AggregateOrganisationReport.id
+            elif not self.report_type and subreport_name_format:
+                parent_report_type = ConcatenatedReport.id
```
Comment (ammar92) [inline rocky/reports/views/base.py:581]: Duplicate code? See lines 583-587

**Review by ammar92 — COMMENTED** [review]
One remark, can be tested in the meantime

File: rocky/reports/views/base.py (hunk 1 above)
Comment (madelondohmen) [reply rocky/reports/views/base.py]: Fixed!

**Review by underdarknl — APPROVED** [review]

**Pull request merged**